### DISCUSSION TITLE
braw files able to replace already existing proxies on my time line ?

### DISCUSSION THREAD

**Discussion by chrystelenicot** [latenitefilms/BRAWToolbox#168] [Support] [answered]
Hello,

I've been editing my project on Braw proxies I did on davinci.
Now I discovered Brawtool box and it changed a bit my way to think the workflow of my project.

I have a lot of chromakey to remove in my images. I'm planning to do it through FCPX (hawaiki tool that works great) and I'm used to do it this way. However I have to send this project at some point to colograding (davinci) so that's why im thinking to use toolbox to have raw file directly sent over davinci to simplify the process without quality loss.

I have a question in this regard;

- Does the braw files imported through toolbox will be easily linked, and be able to replace my proxies clips of my already existing time line  ? Without having to do it manually (i have hundred of clips and effect applied :@)

Many thanks in advance,
C.

**Comment by chrystelenicot** [comment]
I'm adding that I havent buy tool box (yet), I want to be sure it will be usefull in my case and my process :)

**Comment by latenitefilms** [comment, marked as answer]
You can definitely import BRAW files directly/natively into Final Cut Pro using BRAW Toolbox.

You can definitely send these BRAW Toolbox clips to DaVinci Resolve via the BRAW Toolbox Workflow Extension (see [here](https://brawtoolbox.io/toolbox/#convert-project-to-resolve-friendly-fcpxml)).

So yes, you can do all your offline editing in Final Cut Pro and send to DaVinci Resolve.

However, if you're already editing with proxies, I don't think there's any benefit of moving to BRAW files. It would just complicate things and potentially break your workflow.

There is a toolbox to **Add BRAW Toolbox to Proxy Clips within an EVENT**, however it has some [issues](https://brawtoolbox.io/toolbox/#add-braw-toolbox-to-proxy-clips-within-an-event).

My suggestion would be, if you're already editing with proxies, just keep editing with proxies. If you want to use BRAW Toolbox, try it on your next project when you're starting from scratch.
